### PR TITLE
Upgrade everything (pretty much) and add CI

### PR DESCRIPTION
**Pull request by kevindew**
Trello: https://trello.com/c/H49ZOsu1/972-update-all-govspeak-dependencies-in-govuk-apps

This bumps govspeak to resolve [CVE-2020-14001](https://github.com/advisories/GHSA-mqm2-cgpr-p4m6), it also bumps Rails to 6, modernises govuk_publishing_components, modernises rubocop configuration and adds a CI process since there was none before.

### CODE REVIEW

**Review by thomasleese — APPROVED** [review]

File: Rakefile [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
+  require "rubocop/rake_task"
+  RuboCop::RakeTask.new
+rescue LoadError
+  # running in an environment without the rubocop gem enabled
+end
 
 GovspeakPreview::Application.load_tasks
+
+Rake::Task[:default].clear
+task default: %i[rubocop spec]
```
Comment (karlbaker02) [inline Rakefile:17]: Did you mean to change the permissions of the Rakefile?

**Review by karlbaker02 — APPROVED** [review]
Very impressive 🥇 One small comment

File: Rakefile (hunk 1 above)
Comment (kevindew) [reply Rakefile:17]: Yup, it was set as executable before which seemed unconventional and unncessary.

Comment (karlbaker02) [reply Rakefile:17]: I think the permissions got changed back again in the next commit though

Comment (kevindew) [reply Rakefile:17]: Argh - thanks! will sort.

**Pull request merged**